src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubAttribute.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using Microsoft.Azure.WebJobs.Description;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
```

Comment:
Just like the "UseEventHubs" attribute should be in the root namespace, so should the core attributes